### PR TITLE
Crew Pods

### PR DESCRIPTION
**Pull request by Nickvr628**
Allows you to use the game panel to spawn crew pods for ghosts to rejoin the game. It can be clicked on by ghosts to be activated, and it notifies all ghosts when the pod is created. Currently there are Assistant, Engineering, Medical, and Security crew pods purchasable from stations.

https://www.youtube.com/watch?v=9YiV-F0ipiI&feature=youtu.be

:cl: Crew Pods
/adds Crew Pod purchasable from stations.
/:cl:

### CODE REVIEW

**Comment by Stealthkibbler** [comment]
You can have them spawn with ids and other stuff

refer to https://github.com/tgstation/tgstation/blob/master/code/modules/ruins/lavaland_ruin_code.dm for examples

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1651,6 +1651,30 @@
 	contains = list(/obj/item/weapon/book/space_catalog)
 	crate_name = "space catalog crate"
 
+/datum/supply_pack/misc/sssistant
```
Comment (Stealthkibbler) [inline code/modules/cargo/packs.dm]: Assistant

Comment (Nickvr628) [inline code/modules/cargo/packs.dm]: lol thanks

File: code/game/objects/structures/ghost_crew_spawners.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
-//Assistant Cryo Pod purchasable at stations.
+//Civilian Cryo Pod purchasable at stations.
 /obj/effect/mob_spawn/human/purchasable/assistant
```
Comment (Stealthkibbler) [inline code/game/objects/structures/ghost_crew_spawners.dm]: Needs to be changed to Civilian since you changed the list on line 1657

Comment (Nickvr628) [inline code/game/objects/structures/ghost_crew_spawners.dm]: fixed

**Pull request merged**